cmd/server.go

```diff
@@ -13,6 +14,19 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	HeaderCacheControl = "Cache-Control"
+	CacheControlValue  = "max-age=600"
```

Comment:
Let me know what value we need to define here as well... in the response

Reply:
public, max-age=2592000, immutable